### ISSUE TITLE
Running npm command on Windows does not generate a compiled folder

### ISSUE THREAD

**Issue by moekraft** [paypal/skipto#25]
In the instructions to compile the JavaScript, you state: 3. After unzipping or downloading the source code into the directory of your choice, switch to that directory and type these commands:

sh sudo npm install grunt-cli -g npm install grunt  
1. You should now have a directory called **compiled** with the necessary files in it.

Since I am working from a Windows cmd window, I dropped the Linux shell command, sh sudo and just ran npm install grunt-cli -g npm install grunt. Npm installs the requested modules, however, the JavaScript is not compiled. 

Here's the output from the npm command. 
c:\skipto\skipto-master>npm install grunt-cli -g npm install grunt
C:\Users\IBM_ADMIN\AppData\Roaming\npm\grunt -> C:\Users\IBM_ADMIN\AppData\Roam
ng\npm\node_modules\grunt-cli\bin\grunt
C:\Users\IBM_ADMIN\AppData\Roaming\npm\npm -> C:\Users\IBM_ADMIN\AppData\Roamin
\npm\node_modules\npm\bin\npm-cli.js
install@0.1.8 C:\Users\IBM_ADMIN\AppData\Roaming\npm\node_modules\install

grunt-cli@0.1.13 C:\Users\IBM_ADMIN\AppData\Roaming\npm\node_modules\grunt-cli
├── resolve@0.3.1
├── nopt@1.0.10 (abbrev@1.0.5)
└── findup-sync@0.1.3 (glob@3.2.11, lodash@2.4.2)

grunt@0.4.5 C:\Users\IBM_ADMIN\AppData\Roaming\npm\node_modules\grunt
├── dateformat@1.0.2-1.2.3
├── which@1.0.9
├── eventemitter2@0.4.14
├── getobject@0.1.0
├── colors@0.6.2
├── rimraf@2.2.8
├── async@0.1.22
├── hooker@0.2.3
├── grunt-legacy-util@0.2.0
├── exit@0.1.2
├── nopt@1.0.10 (abbrev@1.0.5)
├── minimatch@0.2.14 (sigmund@1.0.0, lru-cache@2.6.4)
├── lodash@0.9.2
├── glob@3.1.21 (inherits@1.0.0, graceful-fs@1.2.3)
├── coffee-script@1.3.3
├── underscore.string@2.2.1
├── iconv-lite@0.2.11
├── findup-sync@0.1.3 (glob@3.2.11, lodash@2.4.2)
├── grunt-legacy-log@0.1.1 (underscore.string@2.3.3, lodash@2.4.2)
└── js-yaml@2.0.5 (esprima@1.0.4, argparse@0.1.16)

npm@2.10.0 C:\Users\IBM_ADMIN\AppData\Roaming\npm\node_modules\npm

Thanks,
Moe

**Comment by moekraft** [comment]
P.S. Tried to running grunt from the command line but it is not found. Does not look to be added to the path after running npm install grunt. Debugging further. Will let you know when I have it resolved. Thanks, Moe

**Comment by moekraft** [comment]
Ok, got it. 
1. First I needed to run cmd as an Administrator using Ctrl + Shift + Enter. See http://www.howtogeek.com/howto/windows-vista/run-a-command-as-administrator-from-the-windows-vista-run-box/
2. I had to manually install dependent grunt plugins, grunt-contrib-less, grunt-contrib-jshint, grunt-contrib-concat, grunt-contrib-uglify, grunt-replace, grunt-contrib-copy, grunt-contrib-compress and grunt-contrib-clean via npm command, e.g. npm install grunt-contrib-less.
3. Manually ran grunt command to compile.
Moe
